From d7d6d769b510118e40d9c0919317665c4c9feb8d Mon Sep 17 00:00:00 2001
From: Alexander Scheel <alexander.m.scheel@gmail.com>
Date: Wed, 12 Jun 2019 17:17:45 -0400
Subject: [PATCH] Disable buffer-based tests

Signed-off-by: Alexander Scheel <alexander.m.scheel@gmail.com>
---
 cmake/JSSTests.cmake | 33 ---------------------------------
 1 file changed, 33 deletions(-)

diff --git a/cmake/JSSTests.cmake b/cmake/JSSTests.cmake
index b389be5c..473d7d4d 100644
--- a/cmake/JSSTests.cmake
+++ b/cmake/JSSTests.cmake
@@ -78,24 +78,6 @@ macro(jss_tests)
         NAME "BigObjectIdentifier"
         COMMAND "org.mozilla.jss.tests.BigObjectIdentifier"
     )
-    jss_test_java(
-        NAME "JSS_Test_PR_FileDesc"
-        COMMAND "org.mozilla.jss.tests.TestPRFD"
-    )
-    jss_test_java(
-        NAME "JSS_Test_Raw_SSL"
-        COMMAND "org.mozilla.jss.tests.TestRawSSL" "${RESULTS_NSSDB_OUTPUT_DIR}"
-        DEPENDS "Setup_DBs"
-    )
-    jss_test_java(
-        NAME "JSS_Test_Buffer"
-        COMMAND "org.mozilla.jss.tests.TestBuffer"
-    )
-    jss_test_java(
-        NAME "JSS_Test_BufferPRFD"
-        COMMAND "org.mozilla.jss.tests.TestBufferPRFD" "${RESULTS_NSSDB_OUTPUT_DIR}" "${DB_PWD}"
-        DEPENDS "List_CA_certs"
-    )
     if ((${Java_VERSION_MAJOR} EQUAL 1) AND (${Java_VERSION_MINOR} LESS 9))
         jss_test_java(
             NAME "Test_PKCS11Constants.java_for_Sun_compatibility"
@@ -126,16 +108,6 @@ macro(jss_tests)
         NAME "JUnit_UTF8StringTest"
         COMMAND "org.junit.runner.JUnitCore" "org.mozilla.jss.tests.UTF8StringTest"
     )
-    jss_test_exec(
-        NAME "buffer_size_1"
-        COMMAND "${BIN_OUTPUT_DIR}/buffer_size_1"
-        DEPENDS "generate_c_buffer_size_1"
-    )
-    jss_test_exec(
-        NAME "buffer_size_4"
-        COMMAND "${BIN_OUTPUT_DIR}/buffer_size_4"
-        DEPENDS "generate_c_buffer_size_4"
-    )
     jss_test_java(
         NAME "JUnit_ChainSortingTest"
         COMMAND "org.junit.runner.JUnitCore" "org.mozilla.jss.tests.ChainSortingTest"
@@ -180,11 +152,6 @@ macro(jss_tests)
         COMMAND "org.mozilla.jss.tests.SSLClientAuth" "${RESULTS_NSSDB_OUTPUT_DIR}" "${PASSWORD_FILE}" "${JSS_TEST_PORT_CLIENTAUTH}" "50"
         DEPENDS "List_CA_certs"
     )
-    jss_test_exec(
-        NAME "TestBufferPRFD"
-        COMMAND "${BIN_OUTPUT_DIR}/TestBufferPRFD" "${RESULTS_NSSDB_OUTPUT_DIR}" "${DB_PWD}"
-        DEPENDS "List_CA_certs" "generate_c_TestBufferPRFD"
-    )
     jss_test_java(
         NAME "Key_Generation"
         COMMAND "org.mozilla.jss.tests.TestKeyGen" "${RESULTS_NSSDB_OUTPUT_DIR}" "${PASSWORD_FILE}"
-- 
2.21.0

